File: 1486-xor-operation-in-an-array/1486-xor-operation-in-an-array.js
/**
 * @param {number} n
 * @param {number} start
 * @return {number}
 */
var xorOperation = function(n, start) {
    total=0
    let arr = new Array(n).fill(0)
    let res = arr.map((num, index) => start + num + 2 * index)
    console.log(res)
    for (num of res) {
        total^=num
    }
    return total
};

File: 1486-xor-operation-in-an-array/1486-xor-operation-in-an-array.py
class Solution:
    def xorOperation(self, n: int, start: int) -> int:
        total=0
        arr = [0] * n
        res = [start + num + 2 * i for i, num in enumerate(arr)]
        for num in res:
            total ^= num

        return total
        